INFO     root:logs.py:31 新增地址前，查询sql数据结果为0
DEBUG    urllib3.connectionpool:connectionpool.py:959 Starting new HTTPS connection (1): mytest.ytoglobal.com:8080
DEBUG    urllib3.connectionpool:connectionpool.py:437 https://mytest.ytoglobal.com:8080 "POST /gateway/app/address/save HTTP/1.1" 200 101
INFO     root:logs.py:31 接口响应数据结果为{u'status': 200, u'message': u'OK', u'data': u'\u4fdd\u5b58\u6210\u529f', u'time': u'2021-08-09 16:41:48'}
INFO     root:logs.py:31 新增地址后，查询sql数据结果为1